src/System.Security.Principal/System.Security.Principal.sln

```diff
@@ -1,12 +1,10 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 14
-VisualStudioVersion = 14.0.22823.1
+VisualStudioVersion = 14.0.24720.0
```

Comment:
The version number update is fine. @Chrisboh was in my office when I looked at this and I was just having fun with him.

In general I don't worry about these VS version number updates in the solution files. I don't think we should take PR's to just update those but if someone is updating the solution for another reason then we should just update these versions with them.